lxd/identities.go

```diff
@@ -1518,6 +1518,10 @@ func updateIdentityCache(d *Daemon) {
 		}
 
 		if cacheEntry.AuthenticationMethod == api.AuthenticationMethodTLS {
+			if cacheEntry.IdentityType == api.IdentityTypeCertificateClientPending {
```

Comment:
To avoid mistakes in the future, can we change this to be an explicit allow list of cacheable identities?